cli.js

```diff
@@ -201,12 +201,18 @@ function init(files) {
 function handlePaths(files) {
 	if (files.length === 0) {
 		files = [
+			'!node_modules',
 			'test.js',
 			'test-*.js',
 			'test/*.js'
 		];
 	}
 
+	// ignore all possible test files in node_modules folder
+	files = files.filter(function (file) {
+		return (file.indexOf('node_modules') === -1);
+	});
```

Comment:
This is code is actually before matched files are found.

At this point, `files` is not an actually a list of files, but a list of glob strings (like `"**/*-test.js`).

You may be seeing results on your machine here because your operating system is automatically expanding glob strings for you (I believe that is the case on OSX, but not on Windows).